helm/ph-ee-engine/values.yaml

```diff
@@ -1794,91 +1794,28 @@ ph-ee-connector_slcb:
     # ph_ee_connector_slcbConfig: {}
     #   ph_ee_connector_slcb.yml: |
 
-connector_bulk_closedloop:
-  enabled: false
+connector:
```

Comment:
please rename bulk_connector to bulk processor to avoid confusion

Reply:
This will be addressed later as backlog